Sources/UAPusher/Builder/Campaigns.swift

```diff
@@ -0,0 +1,44 @@
+
+import JSON
+
+/// Campaigns segment of an Urban Airship push notification payload
+/// See: https://docs.urbanairship.com/api/ua/#campaigns-object
+public struct Campaigns: Segment {
+    
+    // MARK: Class fields
+    
+    public let key: String = "campaigns"
+    public var payload: JSON
+    
+    // MARK: Selector values
+    
+    public enum Selector: JSONRepresentable {
```

Comment:
What's your thoughts on having this `enum` opposed to having another init? E.g. `public init(categories: [String])`. I don't feel that strong over one or the other so just wanted to hear your thoughts.

Reply:
I guess it is a leftover from the first PR where the enum was a requirement in the protocol. I have kept it to preserve some kind of similarity in the Segments